docs/openshift-installation.md

```diff
@@ -0,0 +1,18 @@
+# Openshift Installation
+
+This installation method is the recommended way for Openshift users. **Note**: Openshift version must be 4.2 or higher.
+
+The NGINX Ingress Operator is a [RedHat certified Operator](https://connect.redhat.com/en/partner-with-us/red-hat-openshift-operator-certification).
+
+1. In the Openshift dashboard, click in `Operators` > `Operator Hub` in the left menu and use the search box to type `nginx ingress`:
```

Comment:
Note sure if "click in" is correct or common. Click on?

In https://docs.openshift.com/container-platform/4.3/operators/olm-adding-operators-to-cluster.html just "click" is used.